notificationworkerlambda/src/main/scala/com/gu/notifications/worker/models/InvalidTokens.scala

```diff
@@ -0,0 +1,11 @@
+package com.gu.notifications.worker.models
+
+import play.api.libs.json.{Format, Json}
+
+case class InvalidTokens(
+  tokens: List[String]
+)
+
+object InvalidTokens {
+  implicit val invalidTokensJF: Format[InvalidTokens] = Json.format[InvalidTokens]
+}
```

Comment:
APNS provides a token invalidation timestamp that could be use to avoid discarding a token that would have been refreshed since last time it was fetched. I am not sure we need to handle this edge case though

Reply:
Yes I'm hesitating to implement that, I'm worried it might introduce more edge cases